pkgs/by-name/ka/kasmvnc-xvnc/package.nix

```diff
@@ -0,0 +1,125 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  xorgserver,
+  pkg-config,
+  pixman,
+  mesa,
+  autoconf,
+  automake,
+  xorg-autoconf,
+  fontutil,
+  libtool,
+  libX11,
+  libGL,
+  libgcrypt,
+  xtrans,
+  libxkbfile,
+  libXfont2,
+  libXau,
+  libxcvt,
+  libpciaccess,
+  libepoxy,
+}:
+stdenv.mkDerivation rec {
+  pname = "kasmvnc-xvnc";
+  version = "1.3.3";
+  src = fetchFromGitHub {
+    owner = "kasmtech";
+    repo = "KasmVNC";
+    rev = "v${version}";
+    hash = "sha256-1eX6ulDjYjqaZ+VXbu0ogAPpuNHW88uOCfTqKm6Qkso=";
+  };
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    xorg-autoconf
+    fontutil
+    libtool
+    libX11
+    libGL
+    libgcrypt
+    xtrans
+    libxkbfile
+    libXfont2
+    libXau
+    libxcvt
+    libpciaccess
+    libepoxy
+    mesa
+    pkg-config
+    pixman
```

Comment:
except for pkg-config, these are regular libraries and should be in buildInputs.

Set strictDeps = true and mkDerivation will enforce this strict separation; making the build fail if any dep is in the wrong list.